wolfssl/wolfcrypt/curve25519.h

```diff
@@ -86,6 +86,9 @@ enum {
     EC25519_BIG_ENDIAN=1
 };
 
+WOLFSSL_API
+int wc_curve25519(int public_size, byte* public, int private_size, const byte* private);
```

Comment:
please adjust the naming to follow somewhat like ecc here. The function wc_ecc_make_pub from wolfssl/wolfcrypt/ecc.h. Making this wc_curve25519_make_pub()